features.md

```diff
@@ -0,0 +1,86 @@
+HTMLAttributesRenderer
```

Comment:
Nice docs! 

I'd suggest to change the name of the file to something along the lines of "Extension to attach custom html attributes" to make it more clear what it's about. "Features" is so very generic.

You could also describe the feature in a paragraph in the README.

Reply:
Perhaps we rename the file name from `features.md` into `extensions.md`
and within `extensions.md` we link to any available docs for new extensions or just add extension details to a single file?